package.json

```diff
@@ -4,19 +4,6 @@
   "license": "MPL-2.0",
   "repository": "mozilla/voice-web",
   "private": true,
-  "dependencies": {
```

Comment:
these dependancies are used in the dev scripts OR `http-status-codes` which belongs in the `server/package.json` depdendancies